test/runtests.jl

```diff
@@ -1,4 +1,4 @@
-push!(LOAD_PATH, "/home/guilhermebodin/Documents/Github/Dualization.jl/src")
+push!(LOAD_PATH, "/Users/guilhermebodin/Documents/Dualization.jl/src")
 using Pkg
 Pkg.activate(".") # Just to make sure to use JuMP 0.19.1
```

Comment:
The standard approach would be to have a Project.toml requiring JuMP 0.19.1 in the root directory and then to run the tests you can do `julia --color=yes --project=. test/runtests.jl` or simply `julia --project=.` and then `] test`. Then you don't need these 3 lines and you can set up tests to work with travis

Reply:
I know, I was just a bit lazy, now I have created a Travis and a codecov so we can keep with good CI